ragnarok_item_extractor.py

```
@@ -0,0 +1,409 @@
+        print(f"Analisando tabela de armas: {file_path}")
+        
+        try:
+            # Para arquivos .lub compilados, precisamos de uma abordagem diferente
+            # Vamos tentar extrair strings legíveis do arquivo binário
+            with open(file_path, 'rb') as f:
+                content = f.read()
+            
+            # Converte para string tentando diferentes encodings
+            try:
+                text_content = content.decode('utf-8', errors='ignore')
+            except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)